### ISSUE TITLE
[Bug] (1.16) Using Sildur's Vibrant shaders my Minecraft is locked at 30fps, and without shaders it's locked at 60

### ISSUE THREAD

**Issue by Vertrixx** [sp614x/optifine#4202]
## Description of Issue
Using the Optifine preview pack for 1.16 my game is locked at/around 60 fps. With Sildur's shaders it drops to a maximum of 30fps (consistently at/around 30)

## Steps to Reproduce
1. Use Optifine
2. Use shaders
3. Check fps

## OptiFine Version
1.16.1 G2 pre1

## Installation Method
Standard Optifine .jar file installation

## Log Files/Crash Reports

[2020-07-03-4.log](https://github.com/sp614x/optifine/files/4871198/2020-07-03-4.log)

**DO NOT directly copy and paste log contents here on GitHub.**

## F3 Debug Screenshot

![2020-07-03_20 15 40](https://user-images.githubusercontent.com/20017231/86493640-2b5e3a00-bd6a-11ea-9a25-e13a10c03fb3.png)
![2020-07-03_20 16 25](https://user-images.githubusercontent.com/20017231/86493642-2dc09400-bd6a-11ea-8266-33a227915c6e.png)

## Prior Testing
If you've installed standalone:
- Does this happen in Vanilla without OptiFine?
No.

**Comment by Pizzachu22** [comment]
Is the FPS cap changed to "unlimited"? Have you turned off "Smooth FPS" as well?

**Comment by Violainbow** [comment]
Try turning off VSync. This causes your graphics card to send a new frame to your monitor only when it refreshes, and since shaders make your game run slower, instead of 60 fps on a 60 Hz monitor, you get 30 fps on a 60 Hz monitor.

**Comment by Vertrixx** [comment]
> 
> 
> Is the FPS cap changed to "unlimited"? Have you turned off "Smooth FPS" as well?

I've got it set to unlimited, smooth FPS is off.

**Comment by Vertrixx** [comment]
> 
> 
> Try turning off VSync. This causes your graphics card to send a new frame to your monitor only when it refreshes, and since shaders make your game run slower, instead of 60 fps on a 60 Hz monitor, you get 30 fps on a 60 Hz monitor.

Vsync has been off since forever, my maximum FPS is unlimited. I'm not sure why my Minecraft is locked to 60fps since I've got a 144hz monitor and my system is definitely powerful enough to run shaders at 1440p 144hz.

**Comment by shal0l** [comment]
Did you configure your monitor to display 144Hz? It'll default to 60. Sildurs Extreme is running fine on my end...

**Comment by Vertrixx** [comment]
> 
> 
> Did you configure your monitor to display 144Hz? It'll default to 60. Sildurs Extreme is running fine on my end...

Yeah, it seems to be only minecraft that doesn't go to 144fps. I had it working before I installed optifine like a week or two ago, where the game would go to 144fps, but now it's locked to 60. I've tried pretty much everything that I could find online, but I can't seem to fix it.

I'm using Sildur's medium currently.

**Comment by partyp770** [comment]
You should check if your graphics card's software is automatically limiting your framerate in some way.
